src/utils/promise-finally.js

```diff
@@ -0,0 +1,13 @@
+export default function (promise, onFinally) {
```

Comment:
Maybe it would be better to take code from proposal of [Promise.prototype.finally](https://gist.github.com/rbuckton/66918c8491aa335b003c) and rewrite is as independent function?
Proposal:
```
Promise.prototype.finally = function (callback) {
    return this.then(
        value => this.constructor.resolve(callback()).then(() => value),
        reason => this.constructor.resolve(callback()).then(() => this.constructor.reject(reason))
    );
};
```

Free function:
```
function finally (promise, callback) {
    return promise.then(
        value => promise.constructor.resolve(callback()).then(() => value),
        reason => promise.constructor.resolve(callback()).then(() => promise.constructor.reject(reason))
    );
};
```